config/dev.yaml

```diff
@@ -114,7 +114,7 @@ fe:
     # This is the prefix url in template to cdnjs. This allows you to point at
     # and internal mirror for production deployments.
     # Type: str
-    cdnjs_prefix: "//cdnjs.cloudflare.com"
+    cdnjs_prefix: "http://cdnjs.cloudflare.com"
```

Comment:
The problem is that CSP doesn't allow for scheme agnostic URLs.  So when we craft the CSP it needs to know the correct scheme (which of course we never really know because it may be TLS terminated before...)

What do you suggest?

Reply:
Always serve the resource over HTTPS. It is fine to load HTTPS JS from an insecure origin.